api/CcsSso.Core.ServiceOnboardingScheduler/appsettings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Error",
+      "Microsoft.Hosting.Lifetime": "Error"
+    }
+  },
+  "VaultEnabled": false,
```

Comment:
vaultenable value should be committed as true

Reply:
Only using app secrets